### DISCUSSION TITLE
Apply QDarkStyleSheet theme?

### DISCUSSION THREAD

**Discussion by ichorid** [Tribler/tribler#6690] [Ideas]
We could "outsource" our theme by using some external, consistent QT theme, e.g. https://github.com/ColinDuquesnoy/QDarkStyleSheet

That one even has native Day-Night switching.
Doing so would neatly solve our recurring theming problems.

![изображение](https://user-images.githubusercontent.com/2509103/147486093-a8fa826e-b510-4e6e-a84c-6e0585957d78.png)
![изображение](https://user-images.githubusercontent.com/2509103/147486164-81a346fc-b435-444b-a5d1-9d51662fc8bd.png)

[more screenshots](https://qdarkstylesheet.readthedocs.io/en/latest/screenshots.html#widgets-and-inputs-fields)

Applying the style to Tribler immediately solves [some](#6692) [problems](#6691 ):
![изображение](https://user-images.githubusercontent.com/2509103/147491624-a3bca224-7184-4022-8455-f4d388d7efc3.png)
![изображение](https://user-images.githubusercontent.com/2509103/147491753-01f8cb96-3411-4e87-9acb-d04a61ffafb4.png)

**Comment by ichorid** [comment]
Tribler in action:
![изображение](https://user-images.githubusercontent.com/2509103/147494173-78828fa7-1c0d-4ec0-a6f6-b5aa1afb224f.png)

![изображение](https://user-images.githubusercontent.com/2509103/147494206-011859b7-2156-4a16-b30e-d4592af22a1f.png)

**Reply by devos50** [reply]
Looks ok. Even though it still requires some tweaking with colors/element positions, it can help us to define one unified style. What does light mode look like?

**Reply by ichorid** [reply]
The light one will require much more work to properly integrate with Tribler. Many of our colours are hard-coded, unfortunately.

![изображение](https://user-images.githubusercontent.com/2509103/147568090-15a4c2c2-7f0e-488a-b150-63910f8df611.png)

**Reply by devos50** [reply]
I'm getting some serious 6.5.x flashbacks seeing Tribler with light colors 😁
